CMakeLists.txt

```diff
@@ -112,11 +112,17 @@ if (PEDANTIC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror -pedantic")
 endif (PEDANTIC)
 
+if (SYSCONFDIR)
+    set(SYSCONFDIR ${SYSCONFDIR})
+else ()
+    set(SYSCONFDIR "${CMAKE_INSTALL_PREFIX}/etc")
+endif ()
+
 add_definitions(-DVERSION="${pekwm_VERSION}"
   -DFEATURES="${pekwm_FEATURES}"
   -DBINDIR="${CMAKE_INSTALL_PREFIX}/bin"
   -DDATADIR="${CMAKE_INSTALL_PREFIX}/share"
-  -DSYSCONFDIR="${CMAKE_INSTALL_PREFIX}/etc/pekwm")
+  -DSYSCONFDIR="${SYSCONFDIR/pekwm")
```

Comment:
Nope. '}' still missing here :D

Reply:
Yea, I should switch to a different coffee brand, as current one is not compatible with code reading ;p
need to figure out if I can squash this local with previous without pushing the typo commit first, 
any hints is much appreciated?